packages/sn-document-viewer-react/src/components/ConfirmationDialog.tsx

```diff
@@ -0,0 +1,50 @@
+import Button from '@material-ui/core/Button'
```

Comment:
This is a common scenario that can be reused in other packages as well. Shouldn't we move it into a more "common" package in the future?